### PR TITLE
Added my profile data for the link to be generated

### PR DESCRIPTION
**Pull request by Jaideep25-tech**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2077"></a>

### CODE REVIEW

File: public/data/jaideepSingh.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/jaideep-singh-26a226208/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/jaideep.java/",
+      "icon": "instagram"
+    }
```
Comment (Panquesito7) [inline public/data/jaideepSingh.json:21]: Missing comma.
```suggestion
    },
```

File: public/data/jaideepSingh.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Became the program manger of the open source program SWOC"
+    },
+    {
+      "title": "Started building SOS 42 under SDE internship at Edventure Club",
+      "date": "May 2022",
+      "icon": "dollar",
+      "color": "golden",
+      "description": "Building cross platform mobile application in flutter using state management and firebase",
```
Comment (Panquesito7) [inline public/data/jaideepSingh.json:61]:
```suggestion
      "description": "Building a cross-platform mobile application in Flutter using state management and Firebase"
```

File: public/data/jaideepSingh.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      "icon": "dollar",
+      "color": "golden",
+      "description": "Building cross platform mobile application in flutter using state management and firebase", 
+    },
+    {
+      "title": "Started working at Rjwada (Startup of IIT Delhi)",
+      "date": "Aug 2022",
+      "icon": "dollar",
+      "color": "golden",
+      "description": "Building cross platform mobile application in flutter integrating AR/VR built in unity into Application and working with API ",
```
Comment (Panquesito7) [inline public/data/jaideepSingh.json:68]:
```suggestion
      "description": "Building cross-platform mobile application in Flutter integrating AR/VR built in Unity into Application and working with APIs"
```

File: public/data/jaideepSingh.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      "date": "Nov 2020",
+      "icon": "trophy",
+      "color": "golden"
+    },
+    {
+      "title": "Program Manager",
+      "date": "NOV 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Became the program manger of the open source program SWOC"
```
Comment (Panquesito7) [inline public/data/jaideepSingh.json:54]: Spelling error.
```suggestion
      "description": "Became the program manager of the open source program SWOC"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — COMMENTED** [review]
Hey, 
Thank you for raising the PR and for your contribution. 

Sad to say we can't merge this PR, as we have done huge changes to this repo ([**Migrated to Next.js**](https://github.com/EddieHubCommunity/LinkFree/releases/tag/v1.1.0)), and the path to profile addition has been changed. So please raise a new PR and we will try to merge it ASAP. 

You can find the updated instruction with the new path in the [README](https://github.com/EddieHubCommunity/LinkFree#-contributing). If you have any doubts feel free to reach us out on the [EddieHub Discord](http://discord.eddiehub.org/)

NOTE: Fetch/sync/pull new changes then go for the PR.

**Thank you**

**Pull request closed without merging**